src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -86,6 +96,14 @@ protected void beforeInvoke(Method method, Object[] arguments) {
         waitForAngularIfNeeded(method);
     }
 
+    public void analyzeLog(){
+        List<LogEntry> entries = driver().manage().logs().get(LogType.PERFORMANCE).getAll();
```

Comment:
Why is this method here in this PR?
I actually like the ability to access the browser logs. But please put it in a separate PR. And the result should probably be sent to the test result page(as `ArrayList`?)  instead of standard out. And the ability to make assertions about the content of the logs would probably be good. 
(I've actually had the question whether we could make assertions on the absence of JavaScript errors on pages a couple of times...)

Reply:
That should not be in the PR. My bad. Gonna remove it and maybe will be an separate PR if I finish it.

Reply:
Removed it.